trace-recorder/src/main/org/jetbrains/lincheck/trace/recorder/TraceCollectingEventTracker.kt

```diff
@@ -146,11 +146,14 @@ class TraceCollectingEventTracker(
     private val className: String,
     private val methodName: String,
     private val traceDumpPath: String?,
-    private val mode: TraceCollectorMode
+    private val mode: TraceCollectorMode,
+    private val combine: Boolean
```

Comment:
Maybe rename `combine` to something like `pack` or `packTrace` ? BTW, `pack` is used everywhere else.